presto-native-execution/CMakeLists.txt

```diff
@@ -119,6 +121,19 @@ set(VELOX_BUILD_TEST_UTILS
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+# Set this for backwards compatibility, will be overwritten in velox/
+set(VELOX_GTEST_INCUDE_DIR "velox/third_party/googletest/googletest/include")
+include_directories(.)
+include_directories(${CMAKE_BINARY_DIR})
+include_directories(velox)
+include_directories(velox/velox/external/xxhash)
+include_directories(${VELOX_ROOT_BIN})
+# Before doing anything else, we add velox and any includes it might need. This
+# allows us to bundle the same Find modules that are shipped with velox, but
+# with our own modifications. For instance, our modules can detect if the target
+# dependency is already defined.
+add_subdirectory(velox)
```

Comment:
This allows us to process all of velox's Find modules first before loading ours. Because both velox and presto depend on re2, we process the Find module for re2 twice. However, the upstream one is written in a way that complains if you use it twice (unlike other Find modules in the velox repo). I provide a copy of re2 with a patch in this repo that fixes it.